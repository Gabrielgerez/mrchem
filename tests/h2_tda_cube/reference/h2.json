{
  "input": {
    "constants": {
      "angstrom2bohrs": 1.8897261246257702,
      "dipmom_au2debye": 2.5417464739297717,
      "electron_g_factor": -2.00231930436256,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            1.4
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 7,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -7
    },
    "printer": {
      "file_name": "h2",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {
      "excited_states": {
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 1e-05,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 1e-05
          }
        },
        "run_tda": true,
        "states": [
          {
            "exc_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "final_prec": 1e-05,
              "helmholtz_prec": -1.0,
              "kain": 5,
              "max_iter": 100,
              "method": "Hartree-Fock",
              "orbital_thrs": 0.0001,
              "orth_prec": 1e-14,
              "property_thrs": 0.0001,
              "start_prec": 1e-05
            },
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_exc_0_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_exc_0_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_exc_0_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_exc_0_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_exc_0_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_exc_0_vector.json",
              "file_chk_x": "checkpoint/X_exc_0",
              "file_chk_y": "checkpoint/Y_exc_0",
              "file_x_a": "initial_guess/X_a_exc_0",
              "file_x_b": "initial_guess/X_b_exc_0",
              "file_x_p": "initial_guess/X_p_exc_0",
              "file_y_a": "initial_guess/Y_a_exc_0",
              "file_y_b": "initial_guess/Y_b_exc_0",
              "file_y_p": "initial_guess/Y_p_exc_0",
              "prec": 0.001,
              "type": "cube"
            }
          },
          {
            "exc_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_1",
              "file_chk_y": "checkpoint/Y_rsp_1",
              "final_prec": 1e-05,
              "helmholtz_prec": -1.0,
              "kain": 5,
              "max_iter": 100,
              "method": "Hartree-Fock",
              "orbital_thrs": 0.0001,
              "orth_prec": 1e-14,
              "property_thrs": 0.0001,
              "start_prec": 1e-05
            },
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_exc_1_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_exc_1_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_exc_1_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_exc_1_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_exc_1_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_exc_1_vector.json",
              "file_chk_x": "checkpoint/X_exc_1",
              "file_chk_y": "checkpoint/Y_exc_1",
              "file_x_a": "initial_guess/X_a_exc_1",
              "file_x_b": "initial_guess/X_b_exc_1",
              "file_x_p": "initial_guess/X_p_exc_1",
              "file_y_a": "initial_guess/Y_a_exc_1",
              "file_y_b": "initial_guess/Y_b_exc_1",
              "file_y_p": "initial_guess/Y_p_exc_1",
              "prec": 0.001,
              "type": "cube"
            }
          }
        ],
        "unperturbed": {
          "fock_operator": {
            "coulomb_operator": {
              "poisson_prec": 1e-05,
              "shared_memory": false
            },
            "exchange_operator": {
              "exchange_prec": -1.0,
              "poisson_prec": 1e-05
            },
            "kinetic_operator": {
              "derivative": "abgv_55"
            },
            "nuclear_operator": {
              "proj_prec": 1e-05,
              "shared_memory": false,
              "smooth_prec": 1e-05
            }
          },
          "localize": false,
          "precision": 1e-05
        }
      }
    },
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-05,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-05
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-05,
          "shared_memory": false,
          "smooth_prec": 1e-05
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "mw",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-05,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.7
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 4.122385589338093e-06,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            4.3905426009868324e-12,
            4.280409125783221e-12,
            -4.122385589333533e-06
          ],
          "vector_el": [
            0.0,
            0.0,
            -1.4000041223888764
          ],
          "vector_nuc": [
            4.396384732509632e-12,
            4.286774492689872e-12,
            1.400000000003287
          ]
        }
      },
      "excitation_energies": {
        "state-1": 0.46814789970678633,
        "state-2": 0.4779818529147906
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            1.4
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.5947842934625275
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.189568586925055
      },
      "scf_energy": {
        "E_ee": 1.3169740976622437,
        "E_eext": 0.0,
        "E_el": -1.8480281528851497,
        "E_en": -3.6325425523111403,
        "E_kin": 1.1260549247762603,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337424385994352,
        "E_x": -0.6585146230125135,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 12,
      "routine": "mrchem.x",
      "total_cores": 12,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": {
      "excited_states": {
        "states": [
          {
            "exc_solver": {
              "converged": true,
              "cycles": [
                {
                  "frequency": 0.4680733695359897,
                  "frequency_update": -0.002675797282664091,
                  "mo_residual": 0.015178477625362365,
                  "wall_time": 2.821485847
                },
                {
                  "frequency": 0.46816525231414075,
                  "frequency_update": 9.188277815105199e-05,
                  "mo_residual": 0.004012904019254963,
                  "wall_time": 3.487011511
                },
                {
                  "frequency": 0.4681535082786089,
                  "frequency_update": -1.1744035531832701e-05,
                  "mo_residual": 0.0015299093865147385,
                  "wall_time": 3.942137068
                },
                {
                  "frequency": 0.4681492657665278,
                  "frequency_update": -4.242512081072008e-06,
                  "mo_residual": 0.0009206768553892323,
                  "wall_time": 4.064693883
                },
                {
                  "frequency": 0.468147960978517,
                  "frequency_update": -1.3047880108011686e-06,
                  "mo_residual": 0.00030283676112130293,
                  "wall_time": 4.277915232
                },
                {
                  "frequency": 0.46814788442602323,
                  "frequency_update": -7.655249377393928e-08,
                  "mo_residual": 0.00014420905359179283,
                  "wall_time": 4.220650592
                },
                {
                  "frequency": 0.4681478728254819,
                  "frequency_update": -1.1600541311045877e-08,
                  "mo_residual": 2.1680682348272383e-05,
                  "wall_time": 4.267482502
                }
              ],
              "frequency": 0.46814789970678633,
              "wall_time": 28.862562872
            }
          },
          {
            "exc_solver": {
              "converged": true,
              "cycles": [
                {
                  "frequency": 0.4775863445407528,
                  "frequency_update": -0.002962388527007097,
                  "mo_residual": 0.03000070617647022,
                  "wall_time": 3.175314421
                },
                {
                  "frequency": 0.47799890538491724,
                  "frequency_update": 0.0004125608441644359,
                  "mo_residual": 0.00994573163767513,
                  "wall_time": 3.743217163
                },
                {
                  "frequency": 0.47798399484243775,
                  "frequency_update": -1.4910542479503866e-05,
                  "mo_residual": 0.0024449143473683848,
                  "wall_time": 4.240644596
                },
                {
                  "frequency": 0.4779825580299588,
                  "frequency_update": -1.4368124789184066e-06,
                  "mo_residual": 0.0010094270570627638,
                  "wall_time": 4.371797421
                },
                {
                  "frequency": 0.47798221606969626,
                  "frequency_update": -3.419602625774552e-07,
                  "mo_residual": 0.00028982345067479363,
                  "wall_time": 4.574536821
                },
                {
                  "frequency": 0.47798217874419624,
                  "frequency_update": -3.732550004034321e-08,
                  "mo_residual": 9.013335085724499e-05,
                  "wall_time": 4.575448086
                }
              ],
              "frequency": 0.4779818529147906,
              "wall_time": 26.588227421
            }
          }
        ],
        "success": true
      }
    },
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.3169740976622437,
        "E_eext": 0.0,
        "E_el": -1.8480281528851497,
        "E_en": -3.6325425523111403,
        "E_kin": 1.1260549247762603,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337424385994352,
        "E_x": -0.6585146230125135,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
