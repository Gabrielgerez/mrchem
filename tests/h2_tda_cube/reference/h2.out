                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         excited_states_solver                    ***
***         Git commit hash    3603180aef71c58257df-dirty               ***
***         Git commit author  Gabriel Gerez                            ***
***         Git commit date    Thu Jun 15 18:13:28 2023 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              12
 Total cores             :                                              12
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (12 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 7
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [  128.00000   128.00000   128.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [ -128.00000  -128.00000  -128.00000 ]
 upper bounds          : [  128.00000   128.00000   128.00000 ]
 total length          : [  256.00000   256.00000   256.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       H            :        0.000000        0.000000        1.400000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.700000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project MW molecular orbitals
 Precision               : 1.00000e-04
 Restricted              : True
 MO file                 : initial_guess/phi_p_scf
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              1.000000038380e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-04
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.126054924776
 E-N energy              :            (au)                 -3.632542552311
 Coulomb energy          :            (au)                  1.316974097662
 Exchange energy         :            (au)                 -0.658514623013
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848028152885
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133742438599e+00
                         :      (kcal/mol)             -7.114341213685e+02
                         :        (kJ/mol)             -2.976640363806e+03
                         :            (eV)             -3.085070340020e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.594784293463
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.189568586925
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***           Optimizing Transition Orbitals and Frequencies            ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_exc_0_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.957811825833e-01
===========================================================================


norms of members of X_n: 9.957811e-01
norm of X_n: 1.000000e+00
omega: 4.707492e-01
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize Transition Orbitals and Frequencies
 Guess frequency         : 0.47075 au
 Method                  : Hartree-Fock
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Start precision         : 1.00000e-05
 Final precision         : 1.00000e-05
 Helmholtz precision     : Dynamic
 Property threshold      : 1.00000e-04
 Orbital threshold       : 1.00000e-04
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual     Excited state energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.470749166819          4.707492e-01
lambda: -1.240351e-01
omega: 4.681791e-01
    1          1.517848e-02           0.468073369536         -2.675797e-03
lambda: -1.267109e-01
omega: 4.681561e-01
    2          4.012904e-03           0.468165252314          9.188278e-05
lambda: -1.266190e-01
omega: 4.681499e-01
    3          1.529909e-03           0.468153508279         -1.174404e-05
lambda: -1.266308e-01
omega: 4.681481e-01
    4          9.206769e-04           0.468149265767         -4.242512e-06
lambda: -1.266350e-01
omega: 4.681479e-01
    5          3.028368e-04           0.468147960979         -1.304788e-06
lambda: -1.266363e-01
omega: 4.681479e-01
    6          1.442091e-04           0.468147884426         -7.655249e-08
lambda: -1.266364e-01
omega: 4.681479e-01
    7          2.168068e-05           0.468147872825         -1.160054e-08
omega: 4.681479e-01
last omega: 4.681479e-01
---------------------------------------------------------------------------
                      SCF converged in 7 iterations!
===========================================================================


expectation_values_l: (2.047430e-10,0.000000e+00)
(2.047462e-10,0.000000e+00)
(7.031308e-01,0.000000e+00)
|sum_i <x_i|mu|phi_i>|^2: (4.943929e-01,0.000000e+00)
transition moment (l): (1.542993e-01,0.000000e+00)
expectation_values_v: (0.000000e+00,-1.085172e-10)
(0.000000e+00,-1.085183e-10)
(0.000000e+00,-2.815816e-01)
|sum_i <x_i|v|phi_i>|^2: (7.928818e-02,0.000000e+00)
transition moment (v): (1.129104e-01,0.000000e+00)
===========================================================================
                        Exciation energy (state 1)
---------------------------------------------------------------------------
 Wavelength              :            (nm)                       97.326833
 Frequency               :            (au)                        0.468148
                         :          (cm-1)                   102746.587712
---------------------------------------------------------------------------
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_exc_1_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.974310292951e-01
===========================================================================


norms of members of X_n: 9.974222e-01
norm of X_n: 1.000000e+00
omega: 4.805487e-01
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize Transition Orbitals and Frequencies
 Guess frequency         : 0.48055 au
 Method                  : Hartree-Fock
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Start precision         : 1.00000e-05
 Final precision         : 1.00000e-05
 Helmholtz precision     : Dynamic
 Property threshold      : 1.00000e-04
 Orbital threshold       : 1.00000e-04
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual     Excited state energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.480548733068          4.805487e-01
lambda: -1.142356e-01
omega: 4.780196e-01
    1          3.000071e-02           0.477586344541         -2.962389e-03
lambda: -1.171979e-01
omega: 4.779850e-01
    2          9.945732e-03           0.477998905385          4.125608e-04
lambda: -1.167854e-01
omega: 4.779824e-01
    3          2.444914e-03           0.477983994842         -1.491054e-05
lambda: -1.168003e-01
omega: 4.779819e-01
    4          1.009427e-03           0.477982558030         -1.436812e-06
lambda: -1.168017e-01
omega: 4.779819e-01
    5          2.898235e-04           0.477982216070         -3.419603e-07
lambda: -1.168021e-01
omega: 4.779819e-01
    6          9.013335e-05           0.477982178744         -3.732550e-08
omega: 4.779819e-01
last omega: 4.779819e-01
---------------------------------------------------------------------------
                      SCF converged in 6 iterations!
===========================================================================


expectation_values_l: (1.354082e-05,0.000000e+00)
(1.354082e-05,0.000000e+00)
(6.926582e-04,0.000000e+00)
|sum_i <x_i|mu|phi_i>|^2: (4.801421e-07,0.000000e+00)
transition moment (l): (1.529995e-07,0.000000e+00)
expectation_values_v: (0.000000e+00,-5.931397e-06)
(0.000000e+00,-5.931397e-06)
(0.000000e+00,-2.736132e-04)
|sum_i <x_i|v|phi_i>|^2: (7.493452e-08,0.000000e+00)
transition moment (v): (1.045152e-07,0.000000e+00)
===========================================================================
                        Exciation energy (state 2)
---------------------------------------------------------------------------
 Wavelength              :            (nm)                       95.324440
 Frequency               :            (au)                        0.477982
                         :          (cm-1)                   104904.890967
---------------------------------------------------------------------------
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       H            :        0.000000        0.000000        1.400000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.700000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.126054924776
 E-N energy              :            (au)                 -3.632542552311
 Coulomb energy          :            (au)                  1.316974097662
 Exchange energy         :            (au)                 -0.658514623013
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848028152885
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133742438599e+00
                         :      (kcal/mol)             -7.114341213685e+02
                         :        (kJ/mol)             -2.976640363806e+03
                         :            (eV)             -3.085070340020e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.594784293463
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.189568586925
===========================================================================


===========================================================================
                        Excitation Energies (final)
---------------------------------------------------------------------------
    r                    :                                           Omega
---------------------------------------------------------------------------
    1                    :            (au)                  0.468147899707
    2                    :            (au)                  0.477981852915
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -1.400004
 Magnitude               :            (au)                        1.400004
                         :         (Debye)                        3.558456
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.400000
 Magnitude               :            (au)                        1.400000
                         :         (Debye)                        3.558445
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000       -0.000004
 Magnitude               :            (au)                        0.000004
                         :         (Debye)                        0.000010
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 59s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
